app/serializers/fixed_stream_serializer.rb

```diff
@@ -19,6 +19,11 @@ def serialized_stream(stream)
       unit_symbol: stream.unit_symbol,
       update_frequency: '1 minute',
       last_update: stream.session.last_measurement_at,
+      min: 0,
+      low: 20,
+      middle: 40,
+      high: 60,
+      max: 100,
```

Comment:
should this be included here at the end? The idea was to mock it until the api is not ready

Reply:
Yes, I will delete it once the backend is ready. I left it for testing purposes.